### ISSUE TITLE
Terminal refuses to close itself

### ISSUE THREAD

**Issue by supercomputer7** [SerenityOS/serenity#558]
Hi,
Just noticed while playing with system that sometimes the terminal is stuck completely as a result of exit command. It refuses to close itself apparently.

From the log:
```
<exit> 

<exit> <(null)> 
0x00020752  Process::sys$exit(int) +18
0x0002474e  syscall_trap_entry +818
0x000243d2  syscall_trap_handler +38
0x08069cce  exit +65
0x0804baca  _ZL11run_commandRKN2AK6StringE.part.89 +4535
0x08048671  main +713
0x08048899  _start +94
Finalizing Thread 0 in Shell(128)
Finalizing Process Shell(128)
signal 17 was ignored by Terminal(127)
```
I presume that it happens after long usage so I will try to reproduce that to see if I'm correct.

**Comment by awesomekling** [comment]
Do you have some way to reproduce this? Otherwise I'm not sure what to do with this bug.

**Comment by bugaevc** [comment]
This was probably fixed in 1e1ddce9d8f94dd220174755925cb8d7f03ef36a

**Comment by awesomekling** [comment]
Agreed.
